### DISCUSSION TITLE
[Projects Beta] Triage role is unable to add Issue to Project

### DISCUSSION THREAD

**Discussion by hianhianhian** [community/community#18716] [Projects and Issues] [answered]
Hi Github,

This seems like a possible bug due to the inconsistent behaviour.
In our organisation, we have some users with the Triage role in a certain repository. We use Projects beta in the repo to track issues.

For some reason, the users with Triage role are unable to add issues to the Project from within the Issue side bar. For them, the button is just greyed out and not clickable.

<img width="374" alt="unnamed" src="https://user-images.githubusercontent.com/56030002/174022768-d6c5dddd-ed6f-4793-bf40-b9e2b0032578.png">

However, they are able to add the exact same issue from that repository if they do it from within the project itself.

Could someone have a look at this? Thanks!

**Comment by joe-niland** [comment, marked as answer]
Seems to be a duplicate of https://github.com/github-community/community/discussions/5773
